src/test/java/org/kiwiproject/eureka/junit/EurekaServerExtensionTest.java

```diff
@@ -43,17 +36,17 @@ void shouldCreateTheEurekaServerInstanceAndStartARealServer() throws Exception {
     class AfterAll {
 
         @Test
-        void shouldStopTheServer() throws Exception {
+        void shouldStopTheServer() {
             var extension = new EurekaServerExtension();
             var context = mock(ExtensionContext.class);
 
             extension.beforeAll(context);
             extension.afterAll(context);
```

Comment:
How about an assertion in between that verifies the server is started?

```suggestion
            assertThat(server.isStarted()).isTrue();
            extension.afterAll(context);
```